core/src/test/scala/org/apache/spark/sql/delta/optimize/OptimizeMetricsSuite.scala

```diff
@@ -151,10 +157,10 @@ trait OptimizeMetricsSuiteBase extends QueryTest
 
         spark.sql(s"OPTIMIZE delta.`${tempDir.toString}`") // run optimize on the table
 
-        val actualOperationMetrics = DeltaTable.forPath(spark, tempDir.getAbsolutePath)
-            .history(1)
+        val history = DeltaTable.forPath(spark, tempDir.getAbsolutePath).history(1)
+
+        val actualOperationMetrics = history
             .select("operationMetrics")
```

Comment:
select the operation name here and verify here it self than making another query on the history.

Reply:
Done